_topics/en/multipath-payments.md

```diff
@@ -146,6 +146,18 @@ optech_mentions:
     url: /en/newsletters/2020/05/20/#lightning-loop-using-multipath-payments
     date: 2020-05-20
 
+  - title: "Eclair #1427 and #1439 add support to Eclair for sending multipath payments"
+    url: /en/newsletters/2020/07/08/#eclair-1427
+    date: 2020-07-08
+
+  - title: Eclair 0.4.1 adds support for sending multipath payments
+    url: /en/newsletters/2020/07/08/#eclair-0-4-1
+    date: 2020-07-08
+
+  - title: Zap 0.7.0 Beta adds support for multipath payments
+    url: /en/newsletters/2020/07/22/#zap-0-7-0-beta-released
+    date: 2020-07-22
+
```

Comment:
Suggest we move the "C-Lightning #3809 adds support for sending of multipath payments" item added earlier this month to follow our chronological order.